### ISSUE TITLE
Google Indexing All Notion.so pages

### ISSUE THREAD

**Issue by Javo2804** [stephenou/fruitionsite#61]
Hello! I don't know if someone is having this same issue.

Google is Indexing all Notion.so pages inside my domain, for example: if I my domain is example.com, Google is indexing example.com/customers (and its content is notion.so/customers).
For now, my workaround has been adding those specific urls into my Cloudflare Workers routes without a specifying any Worker.

Does anybody know how can I fix this without all this hassle?

Thanks in advance.
Javier.

**Comment by teddychan** [comment]
You have such a setting in notion.

![image](https://user-images.githubusercontent.com/1013507/99537383-91302300-29e6-11eb-8e20-4f59d7e5c5f8.png)

**Comment by Javo2804** [comment]
Oh, I actually have that option turned on, but it is not what I meant. Is just that when I looked my site up on Google, some results were not from my notion site. For example, if my domain was "example.com", in the results, there were some links like "example.com/careers". Those don't belong to my Notion site, but it does belong to the Notion.com domain.

(Just to be clear, my problem is not that those sites can be accessed from my domain, but that they appear on Google as part of my domain). Sorry if I wasn't clear at the beggining.

Anyways. I found a workaround, so for anyone looking a solution for this: I used [Google Search Console](https://search.google.com/search-console/welcome?hl=es), and manually requested the removal of each one of those links using the [Removals Tool](https://support.google.com/webmasters/answer/9689846).
